### PR TITLE
New tool version format with support for multiple inputs and more

### PR DESCRIPTION
**Pull request by emanueldima**

### CODE REVIEW

**Comment by andmor-** [comment]
Switchboard beta is ready to merge this

**Comment by emanueldima** [comment]
We're now ready to merge, yes.

File: schemas/spec-v2.schema.json [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+        "contact",
+        "inputs"
+    ],
+    "additionalProperties": false,
+    "properties": {
+        "formatVersion": {
+            "type": "string",
+            "enum": ["2"]
+        },
+        "task": {
```
Comment (andmor-) [inline schemas/spec-v2.schema.json:25]: So it does become a free string in schema v2? [see](https://github.com/clarin-eric/switchboard-tool-registry/pull/81#issuecomment-736386849)

File: schemas/validate-all-tools.py [hunk 2]
```diff
@@ -7,8 +7,8 @@
 import fastjsonschema
 
 
-HELP = """Usage: validate-all-tools.py -s <SCHEMA_DEFINITION_FILE> -t <TOOLS_DIRECTORY>"""
-SCHEMA_DEFINITION_FILE = None
+HELP = """Usage: validate-all-tools.py -s <SCHEMA_DEFINITION_FILE1> -t <TOOLS_DIRECTORY>"""
```
Comment (andmor-) [inline schemas/validate-all-tools.py:10]: A suggestion could be to allow -s to load an entire directory of schemas adding the option of passing multiple schemas via a single parameter

**Review by andmor- — COMMENTED** [review]
I am still not sure about a free string for "task". I believe that if this is not in place, we can potentially have a couple of nice simultaneous tasks like "visualization" vs "visualisation", (among other "task proliferation" causes).

File: schemas/spec-v2.schema.json (hunk 1 above)
Comment (emanueldima) [reply schemas/spec-v2.schema.json:25]: I'll copy the list from v1 then.

File: schemas/validate-all-tools.py (hunk 2 above)
Comment (emanueldima) [reply schemas/validate-all-tools.py:10]: Yes, but I don't think we're going to have too many schemas and that sounds like too much work :) I'd say it's ok as it is now.

File: schemas/spec-v2.schema.json (hunk 1 above)
Comment (emanueldima) [reply schemas/spec-v2.schema.json:25]: done in the new commit

**Review by andmor- — APPROVED** [review]

**Pull request merged**